src/main/scala/com/horizon/exchangeapi/BusinessRoutes.scala

```diff
@@ -381,20 +381,17 @@ trait BusinessRoutes extends JacksonSupport with AuthenticationSupport {
       ]
     }
   ],
-  "secretBinding": [
-     {
-       "serviceOrgid": "string",
-        "serviceUrl": "string",
-        "serviceArch": "amd64",
-        "serviceVersionRange": "x.y.z",
-         "secrets": [
-          { 
-           "FirstSecret": "secret1" 
-           "Foo": "Bar"
-          }
-         ]
-      },
-   ],
+"secretBinding": [
+  {
+    "serviceOrgid": "myorg",
+    "serviceUrl": "myservice",
+    "serviceArch": "amd64",
+    "serviceVersionRange": "x.y.z",
+    "secrets": [
+  { "MyServiceSecretName": "MyVaultSecretName" },
```

Comment:
Fix indenting for this line, and the same line in all of the other sections like this. (Use spaces, not tabs)